windows_bat/PokemonGo-Bot-Install.bat

```diff
@@ -203,14 +201,9 @@ if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
 if exist %PGBotPath% rmdir %PGBotPath% /s /q
 if not exist %PGBotPath% md %PGBotPath%
 cd C:\Python27\
-pip2 install --upgrade virtualenv
+pip2 install --upgrade pip
 git clone --recursive -b master https://github.com/PokemonGoF/PokemonGo-Bot %PGBotPath%
-if "%OS%" == "32-BIT" pip2 install --upgrade %PGBotPath%\windows_bat\PyYAML-3.11-cp27-cp27m-win32.whl
-if "%OS%" == "64-BIT" pip2 install --upgrade %PGBotPath%\windows_bat\PyYAML-3.11-cp27-cp27m-win_amd64.whl
-cd %PGBotPath%
-virtualenv .
-call "%PGBotPath%\Scripts\activate.bat"
-pip2 install --upgrade -r %PGBotPath%\requirements.txt

```

Comment:
Installation of setuptools and xxhash works fine, just upgrading it wont work. Virtualenv is broken on some systems because it tries to update setuptools at start too.